pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -149,3 +150,10 @@ type OperatorConfigurationUsers struct {
 }
 
 type Duration time.Duration
+
+type OperatorLogicalBackupConfiguration struct {
+	EnableLogicalBackup      bool   `json:"enable_logical_backup,omitempty"`
```

Comment:
Hmm we can remove the "logical_backup" here or? Given it is already within the struct and the nested key?